main.cpp

```diff
@@ -79,43 +87,84 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+bool allowedToDrinkAlcohol (bool adult, bool insideBar, bool openHours)
+{
+    ignoreUnused (adult, insideBar, openHours);
+    return {};
+}
 
 /*
  2)
  */
-
+bool boilWater (int waterTemperature, int boilingTemperature = 100)
+{
+    ignoreUnused (waterTemperature, boilingTemperature);
+    return {};
+}
 /*
  3)
  */
-
+int parcelCounterDeliver (int john, int max, int gabriel)
+{
```

Comment:
- [ ] revise, pick a function name that represents an action and contains a verb